### PR TITLE
50226_display_all_book_requests_admin

### PR DESCRIPTION
**Pull request by zenith19**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
 
+  namespace :admin do
+  get 'requests/index'
+  end
```
Comment (sujoydatta) [inline config/routes.rb]: remove

File: app/controllers/admin/requests_controller.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Admin::RequestsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+
+  def index
+    @requests = Request.all.page(params[:page]).per 10
```
Comment (MuhammadTamzid) [inline app/controllers/admin/requests_controller.rb]: @requestsall.page(params[:page]).per 10

**Comment by sujoydatta** [comment]
ok

**Comment by MuhammadTamzid** [comment]
Ok

**Comment by sujoydatta** [comment]
ok

File: app/controllers/admin/requests_controller.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class Admin::RequestsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+
+  def index
+    @requests = @requests.all.page(params[:page]).per 10
```
Comment (dieunb) [inline app/controllers/admin/requests_controller.rb]: no need `all`

Comment (MuhammadTamzid) [inline app/controllers/admin/requests_controller.rb]: @requests = @requests.page(params[:page]).per 10

**Pull request merged**